third_party/move/move-compiler-v2/src/pipeline/livevar_analysis_processor.rs

```diff
@@ -53,6 +53,15 @@ pub struct LiveVarInfoAtCodeOffset {
 }
 
 impl LiveVarInfoAtCodeOffset {
+    /// Returns the locals that lives before the program point
+    /// and dies after
+    pub fn released_temps(&self) -> impl Iterator<Item = TempIndex> + '_ {
+        self.before
```

Comment:
I think this could be implemented in linear time rather than (slightly worse than) quadratic. Not sure if it is worth doing the optimization right now, but perhaps add a note about the same?

Reply:
Yes, certainly can make it linear. I've added a todo for this.